### ISSUE TITLE
unable to merge into wiki

### ISSUE THREAD

**Issue by markbirbeck** [Andrew-Chen-Wang/github-wiki-action#2]
I'm not quite sure what to make of this error message:

>  ! [rejected]        master -> master (fetch first)
> error: failed to push some refs to '***github.com/sede-x/dsx-monorepo.wiki.git'
> hint: Updates were rejected because the remote contains work that you do
> hint: not have locally. This is usually caused by another repository pushing
> hint: to the same ref. You may want to first integrate the remote changes
> hint: (e.g., 'git pull ...') before pushing again.
> hint: See the 'Note about fast-forwards' in 'git push --help' for details.
> fatal: could not read Password for 'https://***@github.com': No such device or address

To be honest, I'm not surprised by the first part--about rejecting the push--I expected that. The part that confuses me is that the documentation says that there _must_ be a wiki page in place already. :)

And I'm not sure what to make of the second error message. Did it have any effect on the workflow? It doesn't seem to have, but I'm not sure.

Any help appreciated.

**Comment by Andrew-Chen-Wang** [comment]
To have a Wiki activated, you need to go to the Wiki tab and create a new
page, most likely called Home.md. Then the push will work. For the
GH_TOKEN, are you using the GITHUB_TOKEN secret, and are you using a public
or private repository?

**Comment by markbirbeck** [comment]
Thanks for getting back to me @Andrew-Chen-Wang. In relation to the token, I realised I was doing 'none of the above' due to a typo. ;) I'm using a PAT now, and all is looking good. I might try with `GITHUB_TOKEN` at some point so that it's one less thing to maintain, but it's not urgent.

On the push being rejected, is it possible the rejection was due to the typo in my key, so the error message is incorrect?

The wiki itself was definitely set correctly.

Having said that, during all the different things that I tried, I decided to update the `Home.md` page on the wiki from being blank, to being exactly the same as my local one. So perhaps that was necessary, and then once the token question was resolved, that combination worked.

Anyway...I'm only mentioning all of this in case it helps. Otherwise, it's all looking good now, so thanks again.
